packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -384,6 +402,8 @@ class DefaultTextEditingShortcuts extends Shortcuts {
     SingleActivator(LogicalKeyboardKey.arrowUp, shift: true): DoNothingAndStopPropagationTextIntent(),
     SingleActivator(LogicalKeyboardKey.end, shift: true): DoNothingAndStopPropagationTextIntent(),
     SingleActivator(LogicalKeyboardKey.home, shift: true): DoNothingAndStopPropagationTextIntent(),
+    SingleActivator(LogicalKeyboardKey.end, control: true): DoNothingAndStopPropagationTextIntent(),
```

Comment:
Yeah that might help us avoid a lot of this code.  @gspencergoog Do you have any opinion?

Reply:
We could make a different constructor for Shortcuts that takes a function instead of a Map. I'd want to keep the one that takes a Map, though. One additional benefit of the function over the map would be that it might be faster to construct: long maps can get slow to construct on each build.

Reply:
That would work, so all of this _webShortcuts map would just be a check for kIsWeb and if so return DoNothingAndStopPropagationTextIntent.